Functions.Templates/Functions.Templates.csproj

```diff
@@ -469,6 +500,19 @@
     <Content Include="Templates\TimerTrigger-FSharp\run.fsx" />
     <Content Include="Templates\TimerTrigger-JavaScript\index.js" />
   </ItemGroup>
+  <ItemGroup>
+    <TypeScriptCompile Include="Templates\BlobTrigger-TypeScript\index.ts" />
+    <TypeScriptCompile Include="Templates\EventHubTrigger-TypeScript\index.ts" />
+    <TypeScriptCompile Include="Templates\GenericWebHook-TypeScript\index.ts" />
+    <TypeScriptCompile Include="Templates\GitHubWebHook-TypeScript\index.ts" />
+    <TypeScriptCompile Include="Templates\HttpTrigger-TypeScript\index.ts" />
+    <TypeScriptCompile Include="Templates\ManualTrigger-TypeScript\index.ts" />
+    <TypeScriptCompile Include="Templates\QueueTrigger-TypeScript\index.ts" />
+    <TypeScriptCompile Include="Templates\ServiceBusQueueTrigger-TypeScript\index.ts" />
+    <TypeScriptCompile Include="Templates\ServiceBusTopicTrigger-TypeScript\index.ts" />
+    <TypeScriptCompile Include="Templates\TimerTrigger-TypeScript\index.ts" />
+  </ItemGroup>
```

Comment:
All .ts files should be should have a None Include tag instead of TypeScriptCompile